content/en/docs/reference/config/analysis/ist0105/index.md

```diff
@@ -17,11 +17,11 @@ Istio (which includes the sidecar injector), all running workloads with an Istio
 sidecar must be recreated to allow the new version of the sidecar to be
 injected.
 
-The easiest way to update the sidecar version is to redeploy your application
+To resolve this problem, update the sidecar version by redeploying your application
 using your normal rollout strategy. For a Kubernetes deployment:
 
-* If you're using Kubernetes version 1.15 or above, you can run `kubectl rollout
-  restart <my-deployment>` to trigger a new rollout.
+* If you're using Kubernetes version 1.15 or above, you can run
```

Comment:
```suggestion
* If you're using Kubernetes 1.15 or later, you can run
```

Reply:
I'll suggest using the word "higher" instead. I'm planning to propose that our style guide prescribe using  lower/higher instead of earlier/later older/newer when referring to version numbers, because version doesn't always align to the time dimension. For example, Istio version history where 1.2.7 released after 1.3.2, etc.

Reply:
fixed